File: Assignment_2/request4.py


def main():

    def twoSum(nums, target):

        for i in range(len(nums)):
            if nums[i] < target:
                for j in range(i+1, len(nums)):
                    if nums[i] + nums[j] == target:
                        return ([i,j])

    result = twoSum([2, 11, 7, 15], 9)
    print(result)


if __name__ == '__main__':
	main()

File: Assignment_2/request4.js

function twoSum(nums, target){
    for(i=0; i<nums.length; i++){
        for(j=i+1; j<nums.length; j++){
            if(nums[i] + nums[j] == target){
                if(i != j){
                    return('['+i+', '+j+']')
                }
            }
        }
    }
}

result=twoSum([2, 11, 7, 15], 9)
console.log(result)


